tests/lib/snaps/fork-bomb/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: fork-bomb
+version: 1.0
+license: GPL-3.0
+apps:
+ fork-bomb:
+  command: bin/bomb
```

Comment:
you can add another app called `sh` with a trivial script which does exec and then there'd be no need to install test-snapd-sh (which also pulls in core IIRC)